frontend/src/scenes/authentication/Login.tsx

```diff
@@ -120,20 +146,21 @@ export function Login(): JSX.Element {
                                         Login
                                     </Button>
                                 ) : (
-                                    <Button
-                                        className="btn-bridge"
-                                        data-attr="sso-login"
-                                        htmlType="button"
-                                        block
-                                        onClick={() =>
-                                            (window.location.href = `/login/${precheckResponse.sso_enforcement}/`)
-                                        }
-                                    >
-                                        {SocialLoginIcon(precheckResponse.sso_enforcement)} Login with{' '}
-                                        {SSOProviderNames[precheckResponse.sso_enforcement]}
-                                    </Button>
+                                    <SSOLoginButton
+                                        provider={precheckResponse.sso_enforcement}
+                                        email={form.getFieldValue('email')}
```

Comment:
Someone should convert this to [kea-forms](https://storybook.posthog.net/?path=/docs/how-to-build-a-form--page) 🙃